engine/servermaster/jobmanager.go

```diff
@@ -313,6 +334,25 @@ func (jm *JobManagerImpl) ListJobs(ctx context.Context, req *pb.ListJobsRequest)
 	return resp, nil
 }
 
+// setDetailToMasterMeta sets the results from GetJobDetail to master meta
+func setDetailToMasterMeta(masterMeta *frameModel.MasterMetaKVData, detail []byte, errJob error) {
```

Comment:
This function doesn't make any data persistence, it only constructs a memory value, right?

Reply:
Yes.